spec/Vehicle/Vehicle.vspec

```diff
@@ -150,6 +150,11 @@ VehicleIdentification.KnownVehicleDamages:
   type: attribute
   description: A textual description of known damages, both repaired and unrepaired.
 
+VehicleIdentification.OptionalExtras:
+  datatype:string[]
```

Comment:
I think the syntax require a blank after the colon, likely reason why build fails